lib/client.js

```diff
@@ -91,13 +92,13 @@ class Client {
       apigwSpaceGuid = apiKey.split(':')[0]
     }
 
-    if (!apiKey) {
-      throw new Error(`${messages.INVALID_OPTIONS_ERROR} Missing api_key parameter.`)
+    if (!apiKey && !options.authHandler) {
```

Comment:
SDK uses underscores rather than camelcase for options. Need to consistent with new changes. Change parameter to `auth_handler`?